install_config/cluster_metrics.adoc

```diff
@@ -753,3 +753,291 @@ by performing the following steps:
+
+|`openshift_prometheus_additional_rules_file`
+|Additional Prometheus rules file. Set to `null` by default.
+
+|===
+
+[openshift-prometheus-deploy]
+=== Deploying Prometheus Using Ansible Installer
+
+The Ansible Installer is the default method of deploying Prometheus.
+
+Add label to your node:
+----
+region=infra and deploy prometheus via ansible with default values
+# default value inventory file
+openshift_prometheus_state=present
+openshift_prometheus_namespace=prometheus
+
+openshift_prometheus_replicas=1
+openshift_prometheus_node_selector={"region":"infra"}
+
+# images
+openshift_prometheus_image_proxy="openshift/oauth-proxy:v1.0.0"
+openshift_prometheus_image_prometheus="openshift/prometheus:v2.0.0-dev"
+openshift_prometheus_image_alertmanager="openshift/prometheus-alertmanager:dev"
+openshift_prometheus_image_alertbuffer="ilackarms/message-buffer"
+----
+
+Run the playbook:
+----
+$ ansible-playbook -vvv -i ${INVENTORY_FILE} playbooks/byo/openshift-cluster/openshift-prometheus.yml
+----
+
+[openshift-prometheus--additional-deploy]
+==== Additional Methods for Deploying Prometheus
+
+*Deploy Using Node-Selector*
+
+Label the node on which you want to deploy Prometheus:
+----
+# oadm label node/$NODE ${KEY}=${VALUE}
+----
+
+Deploy Prometheus with Ansible and container resources:
+----
+# Inventory file
+openshift_prometheus_state=present
+openshift_prometheus_namespace=prometheus
+openshift_prometheus_replicas=1
+
+# Set node selector for prometheus
+openshift_prometheus_node_selector={"${KEY}":"${VALUE}"}
+
+# images
+openshift_prometheus_image_proxy="openshift/oauth-proxy:v1.0.0"
+openshift_prometheus_image_prometheus="openshift/prometheus:v2.0.0-dev"
+openshift_prometheus_image_alertmanager="openshift/prometheus-alertmanager:dev"
+openshift_prometheus_image_alertbuffer="ilackarms/message-buffer"
+----
+
+Run the playbook:
+----
+$ ansible-playbook -vvv -i ${INVENTORY_FILE} playbooks/byo/openshift-cluster/openshift-prometheus.yml
+----
+
+*Deploy Using a Non-default Namespace*
+
+Identify your namespace:
+----
+# Inventory file
+openshift_prometheus_state=present
+openshift_prometheus_replicas=1
+openshift_prometheus_node_selector={"region":"infra"}
+
+# Set non-default openshift_prometheus_namespace
+openshift_prometheus_namespace=${USER_PROJECT}
+
+# images
+openshift_prometheus_image_proxy="openshift/oauth-proxy:v1.0.0"
+openshift_prometheus_image_prometheus="openshift/prometheus:v2.0.0-dev"
+openshift_prometheus_image_alertmanager="openshift/prometheus-alertmanager:dev"
+openshift_prometheus_image_alertbuffer="ilackarms/message-buffer"
+----
+
+Run the playbook:
+----
+$ ansible-playbook -vvv -i ${INVENTORY_FILE} playbooks/byo/openshift-cluster/openshift-prometheus.yml
+----
+
+[openshift-prometheus-web]
+==== Accessing the Prometheus Web UI
+
+The Prometheus server automatically exposes a Web UI at `localhost:9090`. You
+can access the Prometheus Web UI with the `view` role.
```

Comment:
Why can this only be access on localhost? 

Why are we not creating a route for this?

Reply:
After deployment, alerts and prometheus routes are created, such as 
alerts route: alerts-prometheus.apps.1109-6kw.qe.rhcloud.com 
prometheus route: prometheus-prometheus.apps.1109-6kw.qe.rhcloud.com

Reply:
@sferich888 
After deployment, alerts and prometheus routes are created, such as 
alerts route: alerts-prometheus.apps.1109-6kw.qe.rhcloud.com 
prometheus route: prometheus-prometheus.apps.1109-6kw.qe.rhcloud.com